oauth/v1/types.go

```diff
@@ -8,7 +8,10 @@ import (
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// OAuthAccessToken describes an OAuth access token
+// OAuthAccessToken describes an OAuth access token.
+// The name of the token object is a sha256 hash of the actual token (not decoded, but plain base64) encoded as base64
+// prefixed with "sha256~" string. It is highly recommended to use base64 with URL and Filename Safe Alphabet (RFC4648)
+// for encoding.
```

Comment:
Not sure what `It is highly recommended to use base64 with URL and Filename Safe Alphabet (RFC4648) for encoding." means.